RedHatInsights/insights-results-aggregator
# implemented cli commands and fixed auth default value

# Description

Fixes #545
Fixes #579
Fixes #580
Fixes #581
Fixes #582
Fixes #583 
Fixes #584

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Refactor (refactoring code, removing useless files)

## Testing steps

`make before_commit` and check output of `./insights-results-aggregator print-config`
